source/hardware/controllers/numark_mixtrack_platinum_fx.rst

```diff
@@ -0,0 +1,830 @@
+|                       |                       |      Sample           |
+|                       |                       |    | Hold and         |
+|                       |                       |      selecting pad,   |
+|                       |                       |      Stop Playback or |
+|                       |                       |      Load Sample if   |
+|                       |                       |      the slot is      |
+|                       |                       |      empty.           |
++-----------------------+-----------------------+-----------------------+
+|    Shift              |    (P9-16) Play       |    | Stop             |
+|                       |    Sample             |      Playback/Load    |
+|                       |                       |      Sample           |
+|                       |                       |    | Hold and         |
+|                       |                       |      selecting pad,   |
+|                       |                       |      Stop Playback or |
+|                       |                       |      Load Sample if   |
+|                       |                       |      the slot is      |
+|                       |                       |      empty.           |
++-----------------------+-----------------------+-----------------------+
+| Alternative           |    | (P1-4) Pitch     |    | Change Range     |
+|                       |      Play             |    | Hold and         |
+|                       |    | Play a single    |      selecting        |
+|                       |      cue Point at     |      SAMPLE, changes  |
+|                       |      variable         |      Range 3 ranges.  |
+|                       |      pitches.         |                       |
+|                       |                       |    | Change cue       |
+|                       |    | *Note*           |    | Hold and         |
+|                       |    | • Only One Cue   |      selecting other  |
+|                       |      at a time •      |      cue point on Cue |
+|                       |      Deleting cue     |      Pad changes cue. |
+|                       |      point in Pitch   |                       |
+|                       |      Play Mode not    |                       |
+|                       |      possible,        |                       |
++-----------------------+-----------------------+-----------------------+
+
+..
+
+**Advanced User Section**
+----------------------
+
+For advanced users, alternative functions exist in the code, by
+accessing and setting a code variable.
+
+**TAPPING**
+Added logic for the tap button, Two modes exist:
+
+The default is to use the MIXXX common bpm.tapButton
+which sets the effective bpm to the one tapped using the 
+tempo adjust. Shift tap resets to 0 tempo change. 
+
+The alternative changes the actual file bpm. The problem 
```

Comment:
This is not a problem, it's the effect of changing the file BPM.

Simply add a hint where users can change the BPM manually, the Track Properties dialog and the BPM column (if inline editing is enabled in Preferences -> Library).

Reply:
Deleted